### DISCUSSION TITLE
[UI/UX] Add descriptions to pull payments

### DISCUSSION THREAD

**Discussion by dstrukt** [btcpayserver/btcpayserver#2586] [General]
Similar to payment requests (which currently has a description field), context is needed for the pull payment.

I'd like to propose a simple description field be added to editor (can be a simple textarea or our wysiwyg) and the end-user view.

Mockups for reference (one with existing editor ui, and potential new create ui) and the end-user view.

Is there anything I'm missing?

Feedback & thoughts welcome!

<img width="623" alt="Screen Shot 2021-06-20 at 6 58 35 AM" src="https://user-images.githubusercontent.com/6250771/122676952-0807e700-d195-11eb-9588-7b7ce069f926.png">
 
<img width="929" alt="Screen Shot 2021-06-20 at 6 41 13 AM" src="https://user-images.githubusercontent.com/6250771/122676947-050cf680-d195-11eb-95a9-bf9c0ae7ef9a.png">

**Comment by bolatovumar** [comment]
Yeah, this seems like a useful thing to have.

**Comment by pavlenex** [comment]
Concept ACK.
